src/mscorlib/src/System/LowLevelConsole.cs

```diff
@@ -10,15 +10,15 @@
 namespace System
 {
     //
-    // Simple limited console class for internal printf-style debugging in mscorlib
-    // and low-level tests that just want to depend on mscorlib.
+    // Simple limited console class for internal printf-style debugging in System.Private.CoreLib
+    // and low-level internal tests that want to call System.Private.CoreLib directly
     //
 
-    public static class Console
+    public static class LowLevelConsole
     {
         static SafeFileHandle _outputHandle;
 
-        static Console()
+        static LowLevelConsole()
         {
             _outputHandle = new SafeFileHandle(Win32Native.GetStdHandle(Win32Native.STD_OUTPUT_HANDLE), false);
```

Comment:
Nit: any reason _outputHandle isn't just initialized at its declaration site?  It could also be readonly.